### PR TITLE
Adding de docs for images

### PR DESCRIPTION
**Pull request by dgabrysch**
I am not sure if creating new files is all that needs to be done to publish localized content, if I need to do something please let me know :)

### CODE REVIEW

**Comment by mkorbi** [comment]
Thank you for your contribution 👍 I will have a look at it soon

File: content/de/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
+
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: I added the tweak, thank you :)

[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: I added the tweak, thank you :)

File: content/de/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+- Prüfen sie ob das Kubelet it dem Parameter `--cloud-provider=aws` läuft.
+- Prüfen sie die Logs des Kubelets (z.B. mit `journalctl -u kubelet`) auf Zeilen wie:
+  - `plugins.go:56] Registering credential provider: aws-ecr-key`
+  - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
+
+### Bei Nutzung der Azure Container Registry (ACR)
+Bei Nutzung der [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/) können sie sich entweder als ein amdministrativer Nutzer, oder als ein Service Principal authentifizieren
+In jedem Fall wird die Authentifizierung über die Standard - Docker Authentifizierung ausgeführt. Diese Anleitung bezieht sich auf das [azure-cli](https://github.com/azure/azure-cli) Kommandozeilenwerkzeug.
+
+Sie müssen zunächst eine Registry erstellen und Authentifizierungsdaten erstellen, eine komplette Dokumentation dazu finden sie hier: [Azure container registry documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli).
```
Comment (mkorbi) [inline content/de/docs/concepts/containers/images.md]: Maybe better ” Sie müssen zunächst eine Registry und Authentifizierungsdaten erstellen,...”

File: content/de/docs/concepts/containers/images.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+Sobald sie ihre Container Registry erstelt haben, nutzen sie die folgenden Authentifizierungsdaten:
+
+   * `DOCKER_USER` : Service Principal oder Administratorbenutzername
```
Comment (mkorbi) [inline content/de/docs/concepts/containers/images.md]: Administrator Passwort

File: content/de/docs/concepts/containers/images.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+cat <<EOF >> ./kustomization.yaml
+resources:
+- pod.yaml
+EOF
+```
+
+Dies muss für jeden Pod getan werden der eine eigene Registry nutzt.
+
+Die Erstellung dieser Sektion kann jedoch automatisiert werden indem man imagePullSecrets einer serviceAccount](/docs/user-guide/service-accounts) Ressource hinzufügt.
+Check [Add ImagePullSecrets to a Service Account](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account) for detailed instructions.
```
Comment (mkorbi) [inline content/de/docs/concepts/containers/images.md]: EN delete

File: content/de/docs/concepts/containers/images.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+Sie können dies in Verbindung mit einer auf jedem Knoten genutzten `.docker/config.json` benutzen, die Zugriffsdaten werden dann zusammengeführt. Dieser Vorgehensweise wird in der Google Kubernetes Engine funktionieren.
+
+
+### Anwendungsfälle
+
+Es gibt eine Anzahl an Lösungen um eigene Registries zu konfigurieren, hier sind einige Anwendungsfälle und empfohlene Lösungen.
+
+1. Cluster die nur nicht-proprietäre Images (z.B. open-source) ausführen. Images müssen nicht versteckt werden.
+   - Nutzing von öffentlichen Images auf Docker hub.
```
Comment (mkorbi) [inline content/de/docs/concepts/containers/images.md]: Docker Hub

File: content/de/docs/concepts/containers/images.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+     - Keine Konfiguration notwendig
```
Comment (mkorbi) [inline content/de/docs/concepts/containers/images.md]: Other list sentence end with a dot, take care to keep one style with/without

**Comment by mkorbi** [comment]
@dgabrysch did you joined us already in the Kubernetes Slack Channel kubernetes-docs-de ? 
Also, we try to track the i18n in this ticket: https://github.com/kubernetes/website/issues/14137
We try to use either one of both to not translate the same page twice :) 

Again, thanks for your contrib :) 👍

**Comment by dgabrysch** [comment]
Not yet but will do :)
I will throw in the changes by tomorrow I guess.

Thank you for reviewing :)

File: content/de/docs/concepts/containers/images.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+### Bei Nutzung der Azure Container Registry (ACR)
+Bei Nutzung der [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/) können sie sich entweder als ein amdministrativer Nutzer, oder als ein Service Principal authentifizieren
+In jedem Fall wird die Authentifizierung über die Standard - Docker Authentifizierung ausgeführt. Diese Anleitung bezieht sich auf das [azure-cli](https://github.com/azure/azure-cli) Kommandozeilenwerkzeug.
+
+Sie müssen zunächst eine Registry erstellen und Authentifizierungsdaten erstellen, eine komplette Dokumentation dazu finden sie hier: [Azure container registry documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli).
+
+Sobald sie ihre Container Registry erstelt haben, nutzen sie die folgenden Authentifizierungsdaten:
+
+   * `DOCKER_USER` : Service Principal oder Administratorbenutzername
+   * `DOCKER_PASSWORD`: Service Principal Password oder Administratorpasswort
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Ok, I think I will let it as it is, googled to be sure: https://www.duden.de/sprachwissen/sprachratgeber/Zusammengesetzte-Substantive
You can and should write "Substantive" together actually :)

File: content/de/docs/concepts/containers/images.md (hunk 3 above)
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Ok, I think I will let it as it is, googled to be sure: https://www.duden.de/sprachwissen/sprachratgeber/Zusammengesetzte-Substantive
You can and should write "Substantive" together actually :)

File: content/de/docs/concepts/containers/images.md [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+   * `DOCKER_REGISTRY_SERVER`: `${some-registry-name}.azurecr.io`
+   * `DOCKER_EMAIL`: `${some-email-address}`
+
+Wenn sie diese Variablen befüllt haben, können sie:
+[configure a Kubernetes Secret and use it to deploy a Pod](/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+
+### Bei Nutzung der IBM Cloud Container Registry
+Die IBM Cloud Container Registry bietet eine mandantenfähige Private Image Registry die Sie nutzen können um ihre Docker Images sicher speichern und teilen können.
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed

File: content/de/docs/concepts/containers/images.md [hunk 9]
```diff
@@ -0,0 +1,344 @@
...
+{{< /note >}}
+
+Dies sind die empfohlenen Schritte um ihre Knoten für eine Nutzung einer eigenen Registry zu konfigurieren, in diesem Beispiel führen sie folgende Schritte auf ihrem Desktop/Laptop aus:
+
+   1. Führen sie `docker login [server]` für jeden Satz ihrer Zugriffsdaten aus.  Dies aktualisiert `$HOME/.docker/config.json`.
+   2. Prüfen Sie `$HOME/.docker/config.json` in einem Editor daruf ob dort nur Zugriffsdaten enthalten sind die Sie nutzen möchten.
+   3. Erhalten sie eine Liste ihrer Knoten:
+      - Wenn sie die Namen benötigen: `nodes=$(kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}')`
+      - Wenn sie die IP - Adressen benötigen: `nodes=$(kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}')`
+   1. Kopieren sie ihre lokale `.docker/config.json` in einen der oben genannten Suchpfade.
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed

File: content/de/docs/concepts/containers/images.md [hunk 10]
```diff
@@ -0,0 +1,344 @@
...
+*   `/.dockercfg`
+
+{{< note >}}
+Eventuell müssen sie `HOME=/root` in ihrer Umgebungsvariablendatei setzen 
+{{< /note >}}
+
+Dies sind die empfohlenen Schritte um ihre Knoten für eine Nutzung einer eigenen Registry zu konfigurieren, in diesem Beispiel führen sie folgende Schritte auf ihrem Desktop/Laptop aus:
+
+   1. Führen sie `docker login [server]` für jeden Satz ihrer Zugriffsdaten aus.  Dies aktualisiert `$HOME/.docker/config.json`.
+   2. Prüfen Sie `$HOME/.docker/config.json` in einem Editor daruf ob dort nur Zugriffsdaten enthalten sind die Sie nutzen möchten.
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed

File: content/de/docs/concepts/containers/images.md [hunk 11]
```diff
@@ -0,0 +1,344 @@
...
+   3. Erhalten sie eine Liste ihrer Knoten:
+      - Wenn sie die Namen benötigen: `nodes=$(kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}')`
+      - Wenn sie die IP - Adressen benötigen: `nodes=$(kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}')`
+   1. Kopieren sie ihre lokale `.docker/config.json` in einen der oben genannten Suchpfade.
+      - Zum Beispiel: `for n in $nodes; do scp ~/.docker/config.json root@$n:/var/lib/kubelet/config.json; done`
+
+Prüfen durch das Erstellen eines Pods der ein eigenes Image nutzz, z.B.:
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed

File: content/de/docs/concepts/containers/images.md [hunk 12]
```diff
@@ -0,0 +1,344 @@
...
+```shell
+kubectl describe pods/private-image-test-1 | grep "Failed"
+  Fri, 26 Jun 2015 15:36:13 -0700    Fri, 26 Jun 2015 15:39:13 -0700    19    {kubelet node-i2hq}    spec.containers{uses-private-image}    failed        Failed to pull image "user/privaterepo:v1": Error: image user/privaterepo:v1 not found
+```
+
+Sie müssen sich darum kümmern das alle Knoten im Cluster die gleiche `.docker/config.json` haben, anderenfalls werden Pods auf einigen Knoten ausgeführt, auf anderen jedoch nicht starten.
+Wenn sie zum Beispiel Knoten automatisch Skalieren lassen, sollte dann jedes Instanztemplate die `.docker/config.json` beinhalten, oder ein Laufwerk einhängen die diese beinhaltet.
+
+Alle Pods haben Lesezugriff auf jedes Image in ihrer eigenen Registry sobald die Registry - Schlüssel zur `.docker/config.json` hinzugfügt wurden.
+All pods will have read access to images in any private registry once private
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Deleted

File: content/de/docs/concepts/containers/images.md [hunk 13]
```diff
@@ -0,0 +1,344 @@
...
+
+```shell
+kubectl describe pods/private-image-test-1 | grep "Failed"
+  Fri, 26 Jun 2015 15:36:13 -0700    Fri, 26 Jun 2015 15:39:13 -0700    19    {kubelet node-i2hq}    spec.containers{uses-private-image}    failed        Failed to pull image "user/privaterepo:v1": Error: image user/privaterepo:v1 not found
+```
+
+Sie müssen sich darum kümmern das alle Knoten im Cluster die gleiche `.docker/config.json` haben, anderenfalls werden Pods auf einigen Knoten ausgeführt, auf anderen jedoch nicht starten.
+Wenn sie zum Beispiel Knoten automatisch Skalieren lassen, sollte dann jedes Instanztemplate die `.docker/config.json` beinhalten, oder ein Laufwerk einhängen die diese beinhaltet.
+
+Alle Pods haben Lesezugriff auf jedes Image in ihrer eigenen Registry sobald die Registry - Schlüssel zur `.docker/config.json` hinzugfügt wurden.
```
[the remark this answers is not in the document]
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed

File: content/de/docs/concepts/containers/images.md (hunk 4 above)
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Deleted & translated

File: content/de/docs/concepts/containers/images.md (hunk 5 above)
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed, it's broken in the original EN docs too: https://kubernetes.io/docs/concepts/containers/images/#use-cases

File: content/de/docs/concepts/containers/images.md (hunk 6 above)
Comment (dgabrysch) [reply content/de/docs/concepts/containers/images.md]: Fixed :)

**Pull request merged**